src/parseTools.js

```diff
@@ -1133,12 +1133,7 @@ function defineI64Param(name) {
 
 function receiveI64ParamAsI32s(name) {
   if (WASM_BIGINT) {
-    // TODO: use Xn notation when JS parsers support it (as of April 6 2020,
-    //  * closure compiler is missing support
-    //    https://github.com/google/closure-compiler/issues/3167
-    //  * acorn needs to be upgraded, and to set ecmascript version >= 11
-    //  * terser needs to be upgraded
-    return `var ${name}_low = Number(${name} & BigInt(0xffffffff)) | 0, ${name}_high = Number(${name} >> BigInt(32)) | 0;`;
+    return `var ${name}_low = Number(${name} & 0xffffffffn) | 0, ${name}_high = Number(${name} >> 32n) | 0;`;
```

Comment:
@kripken, do these `| 0` have any utility these day?   It seems rather pointless give that that we are dealing with BitInit here these should both guaranteed to be integers already, right?